vulnerabilities/templates/base.html

```diff
@@ -49,5 +63,4 @@
     </footer>
 </body>
 {% block javascript %}
-{% endblock %}
-</html>
```

Comment:
No more <html> tag? Is this valid?

Reply:
Yes this is valid HTML

Reply:
It may be but it's inconsistent with most of the web.

Also, the w3c validator complains about it:

> Consider adding a lang attribute to the html start tag to declare the language of this document.

Reply:
This html file does not have a starting html tag, but had a closing one, either we could remove the closing tag (that's what I have done right now) or add an opening html tag.

Reply:
> This html file does not have a starting html tag

Sounds like a bug to me.

Looking at https://github.com/nexB/vulnerablecode/blob/main/vulnerabilities/templates/base.html#L1
Someone mixed the DOCTYPE and the <html> tag.

Reply:
I have added an opening html tag now and left the closing tag as it is.

Reply:
You want to remove the `class="has-navbar-fixed-top"` from the DOCTYPE too.

Reply:
Done!